staging/src/k8s.io/api/testdata/v1.26.0/admissionregistration.k8s.io.v1alpha1.ValidatingAdmissionPolicy.after_roundtrip.json

```diff
@@ -126,7 +126,8 @@
         "reason": "reasonValue"
       }
     ],
-    "failurePolicy": "failurePolicyValue"
+    "failurePolicy": "failurePolicyValue",
+    "variables": null
```

Comment:
addition of this should have raised a review flag... in general, we don't want new fields to change serialization of old objects by default

fixed in https://github.com/kubernetes/kubernetes/pull/121000

Reply:
Thank you for pointing out! We will take special attention to this kind of problem in fixtures in future API changes.